Jenkinsfile

```diff
@@ -1,90 +1,118 @@
 #!groovy
 import groovy.json.JsonOutput
+import groovy.json.JsonBuilder
 import groovy.json.JsonSlurper
 
 node {
-	stage('Read test names') {
-        checkout scm
-	    stash name: 'scripts', includes: 'um2junit.rb'
+    stage('Read and Execute tests') {
         // Obtain credentials for accessing the useMango server, which should be stored in Jenkins with the ID of 'usemango'
-	    withCredentials([usernamePassword(credentialsId: 'usemango', usernameVariable: 'user', passwordVariable: 'pwd')]) {
+        withCredentials([usernamePassword(credentialsId: '', usernameVariable: 'user', passwordVariable: 'pwd')]) {
```

Comment:
Fill in a credential name, we'll instruct the user to create credentials of that name. We want this script to be fully ready so the users can use it as is.

The credential type here should just be a simple secret string not a username and password.